.eslintrc

```diff
@@ -1,4 +1,5 @@
 {
+  "parser": "babel-eslint",
```

Comment:
Lots of ES7 syntax was erroniously triggering standard errors, e.g., class properties:
```javascript
class Foo
  static propTypes = { ... }
  foo = () => { ... } 
``` 

By parsing through `babel`, it picks up language settings in `.babelrc`.